docs/WinRTAPIContracts.md

```diff
@@ -0,0 +1,76 @@
+# Windows App SDK API Contracts
```

Comment:
Reads more like a spec than coding guidelines. Mostly it's WHY and little on the HOWTO guide for developers writing code.

SUGGESTION: Move this to `specs\WinRT\WinRTAPIContracts.md` and once mostly baked out add a `docs/Coding-Guidelines/WinRTAPIContracts.md` focused on the HOWTO guidance for devs writing code with a link to the spec for those who want to know WHY